x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/AuthorizationEngine.java

```diff
@@ -241,15 +241,17 @@ void checkPrivileges(
     void getUserPrivileges(AuthorizationInfo authorizationInfo, ActionListener<GetUserPrivilegesResponse> listener);
 
     /**
-     * Retrieve remote access privileges for a given target cluster, from the provided authorization information, to be sent together
+     * Retrieve cross cluster access privileges for a given target cluster, from the provided authorization information, to be sent together
      * with a cross-cluster request (e.g. CCS) from an originating cluster to the target cluster.
      */
-    default void getRemoteAccessRoleDescriptorsIntersection(
+    default void getRoleDescriptorsIntersectionForRemoteCluster(
```

Comment:
Another candidate that comes to mind is `getRoleDescriptorsWithRemotePrivileges` but I'm not sure which one's better

Reply:
I like the current proposal